core/src/main/java/org/infinispan/configuration/serializing/CoreConfigurationSerializer.java

```diff
@@ -409,6 +409,7 @@ private void writeReplicatedCache(ConfigurationWriter writer, String name, Confi
       } else {
          writer.writeMapItem(configuration.isTemplate() ? Element.REPLICATED_CACHE_CONFIGURATION : Element.REPLICATED_CACHE, Attribute.NAME, name);
       }
+      configuration.clustering().hash().attributes().write(writer);
```

Comment:
unfortunately this also serializes `owners` which is invalid for repl caches

Reply:
right! I've fixed it.